bitsv/network/services/bchsvexplorer.py

```diff
@@ -0,0 +1,83 @@
+import requests
+import json
+from decimal import Decimal
+
+from bitsv.network import currency_to_satoshi
+from bitsv.network.meta import Unspent
+from bitsv.network.transaction import Transaction, TxPart
+
+DEFAULT_TIMEOUT = 30
+BSV_TO_SAT_MULTIPLIER = 100000000
```

Comment:
Do we already have a constant for this?

Reply:
Yes we do. It should be imported from one place rather than like this. Good idea.